DTS-STN/secure-client-hub
# Add cachified to all AEM content fetches

### Changelog 
fix:add caching to AEM calls

### Description of proposed changes:
This PR implements cachified to cache each call to AEM in our mappers so data is only refreshed every 10 minutes. This prevents AEM from being overloaded (as we were seeing in our stress tests) and also makes our application much faster.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev` (or build and start)
3. Ensure application runs as expected
